sdk/streamanalytics/Microsoft.Azure.Management.StreamAnalytics/src/Microsoft.Azure.Management.StreamAnalytics.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.StreamAnalytics</PackageId>
 		<Description>Provides streaming job management capabilities for Microsoft Azure.</Description>
 		<AssemblyName>Microsoft.Azure.Management.StreamAnalytics</AssemblyName>
-		<Version>3.0.0-preview</Version>
+		<Version>3.0.0-pure</Version>
 		<PackageTags>Microsoft Azure Stream Analytics management;Stream Analytics management;streaming jobs;streaming job management;</PackageTags>
 		<PackageReleaseNotes>
```

Comment:
I don't think the version `3.0.0-pure` meets the semantic versioning requirement. With some breaking changes in this PR, probably use `4.0.0-preview`. Please also update the `PackageReleaseNotes` section.

Reply:
By the way, since we are going to GA, may be 4.0.0-pure?

Reply:
will add it as 4.0.0